src/styles/header/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+  box-sizing: border-box;
+}
+
+.nav {
+  display: flex;
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    color: #060b35;
+    font-size: 12px;
+    line-height: 60px;
+    font-weight: 500;
+    text-transform: uppercase;
+
+    &:hover {
+      color: var(--primary-color);
+      text-decoration: none;
```

Comment:
The `text-decoration: none;` rule is redundant here as it's already declared for the `&__link` element on line 22. You can safely remove it.